client/src/components/pickWinnerComponents/PickWinnerStyles.js

```diff
@@ -0,0 +1,86 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  //input fields
+  mainBox: {
+    display: 'block',
+    overflow: 'auto',
+    margin: 'auto',
+  },
+
+  container: {
+    width: '100%',
+  },
+
+  innerContainer: {
+    maxWidth: '80%',
+  },
+
+  titleLogo: {
+    flexGrow: 1,
+  },
+
+  title: {
+    flexGrow: 1,
+    textAlign: 'center',
+    marginTop: theme.spacing(5),
+    marginBottom: theme.spacing(5),
+    fontFamily: 'Poppins-Semibold',
+  },
+
+  subtitle: {
+    textAlign: 'left',
+    paddingTop: theme.spacing(6),
+    fontFamily: 'Poppins-Semibold',
+  },
+
+  subtext: {
+    color: theme.palette.grey.dark,
+    paddingTop: theme.spacing(1),
+    position: 'relative',
+  },
+
+  //tattoo grid
+  root: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    justifyContent: 'space-around',
+    overflow: 'hidden',
+    backgroundColor: theme.palette.background.paper,
+  },
+
+  gridList: {
+    flexWrap: 'nowrap',
+    // Promote the list into his own layer on Chrome. This cost memory but helps keeping high FPS.
+    transform: 'translateZ(0)',
+  },
+
+  //create contest button
+  alignItemsAndJustifyContent: {
```

Comment:
CSS selectors should be named for what they are styling, now how they are styling it

Reply:
This one has been removed as I'm not using it, but others are updated.